kubernetes/linera-validator/build_and_redeploy.sh

```diff
@@ -86,17 +86,22 @@ if [ "$cloud_mode" = true ]; then
 
```

Comment:
<i class='graphite__hidden'>[Re: lines 81 to 81]<br/></i>

I tried doing this parametrization before also, but what got me stuck was how to pass the args to `gcloud builds submit` from here, without having to create a bunch of different cloudbuild YAML files (that might be the only solution, but idk).
We need to make sure running this script with cloud build also works with the new Dockerfile<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/linera-io/linera-protocol/1192?utm_source=unchanged-line-comment">Graphite</a>.</span>